packages/core/src/config/types.ts

```diff
@@ -238,6 +238,16 @@ export interface Config<P = APIEvent> {
      */
     enabled?: boolean;
   };
+
+  /**
+   * This is a preview feature.
+   * We have tested it thoroughly, but it is possible that it might not work as expected in all cases.
+   */
+  trackUserActions?: boolean;
+  /**
+   * Configure your own attribute name for tracking user actions. Default is 'data-faro-user-action-name'
+   */
+  trackUserActionsDataAttributeName?: string;
```

Comment:
just adding a new line for style 
```suggestion

  /**
   * Configure your own attribute name for tracking user actions. Default is 'data-faro-user-action-name'
   */
  trackUserActionsDataAttributeName?: string;
```